packages/transformer/src/test/IModelTransformerUtils.ts

```diff
@@ -2136,6 +2142,17 @@ export class RecordingIModelImporter extends CountingIModelImporter {
     };
     return this.targetDb.elements.insertElement(auditRecord);
   }
+  private accountForPartialViewDefinition2d(elementProps: ElementProps): void {
+    const view2d = elementProps as unknown as ViewDefinition2d;
+    // if the ViewDefinition2d has a baseModelId, it should be a valid Id64String otherwise insert/get operations will fail.
```

Comment:
@khanaffan @MichaelSwigerAtBentley, would appreciate reviewing this func, as the issue started after some of you're guys work in native. I think Mike worked with Anmol a bit on this, but wanted a quick review to make sure this logic seems sound. Lgtm, but want to make sure temporarily setting this baseModelId wouldn't have unanticipated consequences before the correct value is reconciled. Thanks!